modules/ROOT/pages/rest-sdk/edits/rest-sdk-connectivity.adoc

```diff
@@ -0,0 +1,53 @@
+= REST SDK
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+REST SDK enables you to create a connector from an API specification (OAS or RAML)
+and uses a metadata-driven approach to accelerate connector development. REST SDK
+provides a quick and relevant connectivity experience that is easy to use and requires
+little to no configuration.
+
+With REST SDK you can:
```

Comment:
I like this! One thing I would like to change is "build times" since I don't want it confused with the actual build used to make connectors. Does "short connector creation times, sometimes from..." sound weird to you? @kbostrom